### PR TITLE
regenerated the template project

### PR DESCRIPTION
**Pull request by kylewalke**
API v55 empty dx project generated.

### CODE REVIEW

File: force-app/main/default/aura/.eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "plugins": ["@salesforce/eslint-plugin-aura"],
```
Comment (kylewalke) [inline force-app/main/default/aura/.eslintrc.json:2]: Do we want to include these lint files?

File: package.json [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  "version": "1.0.0",
+  "description": "Salesforce App",
+  "scripts": {
+    "lint": "eslint **/{aura,lwc}/**",
+    "test": "npm run test:unit",
+    "test:unit": "sfdx-lwc-jest",
+    "test:unit:watch": "sfdx-lwc-jest --watch",
+    "test:unit:debug": "sfdx-lwc-jest --debug",
+    "test:unit:coverage": "sfdx-lwc-jest --coverage",
+    "prettier": "prettier --write \"**/*.{cls,cmp,component,css,html,js,json,md,page,trigger,xml,yaml,yml}\"",
```
Comment (kylewalke) [inline package.json:13]: same with these prettier lines?

File: package.json [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    "prettier:verify": "prettier --list-different \"**/*.{cls,cmp,component,css,html,js,json,md,page,trigger,xml,yaml,yml}\"",
+    "postinstall": "husky install",
+    "precommit": "lint-staged"
```
Comment (kylewalke) [inline package.json:16]: and this one too?

**Review by geomuck — APPROVED** [review]
Talked with Kyle offline and he confirmed that this is what the SFDX CLI produces during an execution of sfdx force:project:create at its latest version, today.

Looks good to me!

**Pull request merged**